### ISSUE TITLE
Metadata patching

### ISSUE THREAD

**Issue by mfansler** [conda-forge/r-knitr-feedstock#35]
As encountered in https://github.com/conda-forge/r-lintr-feedstock/issues/12, the `r-knitr=1.31` builds did not properly declare new minimum bounds for `r-xfun` (should be `>=0.19`). This should be patched in the metadata to prevent future solving issues.

**Comment by jdblischak** [comment]
@mfansler Are you planning on submitting the repodata patch or would you prefer to delegate the task?

**Comment by mfansler** [comment]
@jdblischak thanks for asking - I don't plan to get to this. You or anyone else are welcome to help out!

**Comment by jdblischak** [comment]
I patched r-knitr 1.31 and also all the other versions that require a minimum version of r-xfun in https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/676. We can close this Issue once it is merged

**Comment by jdblischak** [comment]
My patch was merged in https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/commit/24802fda02647a66c965b1ef676d3fdbea10f15f

I confirmed it fixes the problem:

```sh
mamba create --dry-run -n test r-knitr=1.31 'r-xfun<0.19'
## Could not solve for environment specs
## The following packages are incompatible
## ├─ r-knitr 1.31**  is installable and it requires
## │  └─ r-xfun >=0.19 , which can be installed;
## └─ r-xfun <0.19  is not installable because it conflicts with any installable versions previously reported.

mamba create --dry-run -n test r-knitr=1.31 r-xfun=0.19
# solves
```
